ivy/functional/backends/torch/manipulation.py

```diff
@@ -348,3 +348,20 @@ def clip(
 clip.support_native_out = True
 
 clip.unsupported_dtypes = ("float16",)
+
+
+def movedim(
+    x: torch.Tensor,
+    source: int,
```

Comment:
if you look at the torch documentation [here](https://pytorch.org/docs/stable/generated/torch.movedim.html#torch.movedim), you'll see that `source` is of type int or tuple of ints, so you can write it as `Union[int, Tuple[int, ...]]`